### PR TITLE
CNV-27405: VM configuration tab

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.13

Issue: [CNV-27405](https://issues.redhat.com//browse/CNV-27405)

Link to docs preview:

- Web console overview: [Configuration tab](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-web-console-overview.html#ui-virtualmachine-details-configuration_virt-web-console-overview)
- [Viewing QEMU info](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-viewing-qemu-guest-agent-web.html#About-the-qemu-guest-agent-web_virt-viewing-qemu-guest-agent-web)
- [Adding disk to VM](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-edit-vms.html#virt-add-disk-to-vm_virt-edit-vms)
- [Adding disk to template](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-editing-vm-template.html#virt-add-disk-to-vm_virt-editing-vm-template)
- [Adding NIC to VM](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-edit-vms.html#virt-vm-add-nic_virt-edit-vms)
- [Adding NIC to template](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-editing-vm-template.html#virt-vm-add-nic_virt-editing-vm-template)
- [Managing config maps, secrets, service accounts](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-edit-vms.html#virt-adding-secret-configmap-service-account-to-vm_virt-edit-vms)
- [Hot plugging VM disk UI](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.html#virt-hot-plugging-a-virtual-disk-using-the-web-consolevirt-hot-plugging-virtual-disks)
- [Hot unplugging VM disk UI](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.html#virt-hot-unplugging-a-virtual-disk-using-the-web-consolevirt-hot-plugging-virtual-disks)
- [Enabling dedicated resources - VM](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/advanced_vm_management/virt-dedicated-resources-vm.html#virt-enabling-dedicated-resources_virt-dedicated-resources-vm)
- [Enabling dedicated resources - VM template](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-dedicated-resources-vm-template.html#virt-enabling-dedicated-resources_virt-dedicated-resources-vm-template)
- [Generalizing Win VM Sysprep](https://57756--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-automating-windows-sysprep.html#virt-generalizing-windows-sysprep_virt-automating-windows-sysprep)

QE review: 
[ x ] Approved by QE

Additional information: 

Added the Configuration tab, moved Scheduling and other tabs to the Config tab. 
Deleted periods after sentence fragments in tables. Removed some obsolete content.

### CODE REVIEW

**Comment by apinnick** [comment]
@gouyang  Could you please review?

File: modules/virt-add-disk-to-vm.adoc [hunk 1]
```diff
@@ -33,9 +35,9 @@ Use this procedure to add a virtual disk to a {object}.
 
 . Select a {object} to open the *{object-vm-overview}* screen.
 
-. Click the *Disks* tab and then click *Add disk*.
+. On the *{tab}* tab, click *Add disk*.
```
Comment (gouyang) [inline modules/virt-add-disk-to-vm.adoc:38]: The tab value is "Scheduling" from line #22, which is not right as it should be "Disks" tab here, not sure if change line #22 will impact other place, so add a comment here.

File: modules/virt-add-disk-to-vm.adoc [hunk 2]
```diff
@@ -11,13 +11,15 @@ ifeval::["{context}" == "virt-edit-vms"]
 :object: virtual machine
 :object-gui: VirtualMachines
 :object-vm-overview: VirtualMachine details
+:tab: Configuration -> Scheduling
```
Comment (gouyang) [inline modules/virt-add-disk-to-vm.adoc:14]: "Scheduling" should be "Disks".

**Comment by mburke5678** [comment]
@apinnick A few nits. Otherwise LGTM

I know this is existing text, but can you take a look at these uses of the word screen [here](https://github.com/openshift/openshift-docs/pull/57756/files#diff-b96b74629b356ba8a89cf6a6c862fa13a739cab62ae1ca227ba78c4cf66cfb06R36) and [here](https://github.com/openshift/openshift-docs/pull/57756/files#diff-2ffe4c671510dca55c7eb272b35e7a68b55d989b705db7fd885e06a44ff54711R33). Should these say _tab_ instead, to be consistent?

**Pull request merged**